### ISSUE TITLE
Useful Documentation Explicitly Omitted

### ISSUE THREAD

**Issue by Swivelgames** [reactjs/redux#2896]
### Documentation Bug

While perusing the section on `Reselect` and using selectors to create "views" of your reducer state, I noticed a somewhat alarming disclaimer that stopped me in my tracks.

Naturally, I'm utilizing Redux with React. Additionally, I typically have high hopes that the documentation will provide enough information for me to explore the depths of a library or utility and use it optimally.

Unfortunately, when reading about selectors, I was told quite literally that what was about to be shown "requires changes to the [code base] ... and have been omitted for brevity."

Brevity is important in writing documentation, but it's incredibly frustrating when you run upon a disclaimer that essentially says, "The things we're doing in this section? You can't do them without X and Y, but we're not going to tell you how to do X and Y, so good luck." This is less brevity, and instead excluding useful or important information from the documentation. A better solution might be to move that information to another article and reference it from the original.

#### Expected Result
- An additional section, or a link or reference to additional materials for those who would like to do what your documentation is demonstrating.

#### Actual Result
- Disclaimer does not include a link or reference to any explanation of the items in the examples below it.
- OR, disclaimer does not include a link to the example's source code, which could then be used to better understand the additional documentation for those who want to dive deeper.

The most unfortunate part is that I was specifically searching the documentation for this very section in order to understand exactly what I quickly found out was intentionally omitted.

> **Accessing React Props in Selectors**
>
> _This section introduces a hypothetical extension to our app that allows it to support multiple Todo Lists. Please note that a full implementation of this extension requires changes to the reducers, components, actions etc. that aren't directly relevant to the topics discussed and have been omitted for brevity._

(Source: [https://github.com/reactjs/redux/blob/master/docs/recipes/ComputingDerivedData.md](https://github.com/reactjs/redux/blob/master/docs/recipes/ComputingDerivedData.md))

P.s., Keep up the great work!

**Comment by markerikson** [comment]
I'd appreciate it if you could file a PR that expands on this section.  You may want to refer to my post [Idiomatic Redux: Using Reselect Selectors for Performance and Encapsulation](http://blog.isquaredsoftware.com/2017/12/idiomatic-redux-using-reselect-selectors/), as well as some of the articles in the [Redux Reducers and Selectors](https://github.com/markerikson/react-redux-links/blob/master/redux-reducers-selectors.md#selectors) section of my React/Redux links list.

**Comment by hedgerh** [comment]
I took a look at this and wrote out the reducer part.  I'm not 100% finished, but wanted to get some feedback before I went any further to see if this really should be added.  It seems like it's a lot of code, but on the other hand, I think recipes shouldn't gloss over anything.

If you all think this is fine, I'll wrap it up and put in a PR.  Otherwise, maybe we can just show a model of how the state looks in this example, or come up with a less complex example.

https://github.com/hedgerh/redux/pull/1/files

The rendered markdown
https://github.com/hedgerh/redux/blob/d051fdc052e4efd051c4afba5dab65abe90541e5/docs/recipes/ComputingDerivedData.md
